<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertQuery</class>
 <widget class="QWidget" name="InsertQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert</string>
  </property>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
